bin/polyline.bin.js

```diff
@@ -1,43 +1,53 @@
 #!/usr/bin/env node
 
-var polyline = require('../');
+const polyline = require('../');
 
-var HELP = 'Provide data from stdin and use with --decode (default), --encode, --toGeoJSON, or --fromGeoJSON\n';
+const HELP = 'Provide data from stdin and use with --decode (default), --encode, --toGeoJSON, or --fromGeoJSON. Optionally provide a precision with --precision=6\n';
 
-var mode = process.argv[2] || '--decode';
-var rawInput = '';
+let precision = 5;
+let rawInput = '';
+const mode = process.argv[2] || '--decode';
+
+if (process.argv[3]) {
```

Comment:
Does this mean that we are now enforcing an order in the flags passed to the CLI? Is there a way to make them able to be ordered any way the user wants? If not, we should note the required order in the docs

Reply:
Yeahhh good point. I waffled with the idea of using https://www.npmjs.com/package/meow but that would be adding a dependency to a package that doesn't have any. But that would be a huge help in making the commandline arguments flexible.

Reply:
ooooh this looks like a nice tool. I think that you should feel comfy adding this! Maybe seek comment from the project owner if you feel iffy though

Reply:
I'm convinced :) going to add it 👍